src/quicktype-core/language/ruby/index.ts

```diff
@@ -553,7 +555,9 @@ export class RubyRenderer extends ConvenienceRenderer {
 
     private emitTypesModule() {
         this.emitBlock(["module Types"], () => {
-            this.emitLine("include Dry::Types.module");
+            const includeType = this._options.legacyGemVesion ? "Dry::Types.module" : "Dry.Types()";
```

Comment:
Please keep in mind that those two modules are not exactly identical.

Back when we used `Dry::Types.module`, we could access _nominal_ types using `Types::String` and strict types using `Types::Strict::String`.

When we're using `Dry.Types()`, `Types::String` is now the same as `Types::Strict::String` – it's a possibly breaking behavior. See [changelog](https://github.com/dry-rb/dry-types/blob/master/CHANGELOG.md#changed-3)

If we want to keep the old behavior, we should replace it with `Dry.Types(default: :nominal)`

Reply:
```suggestion
const includeType = this._options.legacyGemVesion ? "Dry::Types.module" : "Dry.Types(default: :nominal)";
```

I'm not good in Dry things.
You mean something like this, without any additional configuration?

Reply:
Yes, that'll do the trick